src/hooks/useSearch.ts

```diff
@@ -28,14 +26,14 @@ const useSearch: UseSearch = (projectId, q) => {
     { enabled: Boolean(projectId) && Boolean(q) },
   );
   const issueIds = useMemo(() => {
-    return map(get(searchIssues.data, ["data"], []) || [], "id");
+    return (Array.isArray(searchIssues.data?.data) ? searchIssues.data?.data ?? [] : []).map(({ id }) => id)
```

Comment:
`return (searchIssues.data?.data ?? []).map(({ id }) => id)`

Reply:
I'm not sure about the response because it comes from the API. It seems to be better to recheck.